examples/amp-script/example.amp.html

```diff
@@ -19,8 +19,8 @@
   <script async custom-template="amp-script" src="https://cdn.ampproject.org/v0/amp-script-0.1.js"></script>
 </head>
 <body>
-  <h1>Example: hello-world.js</h1>
-  <amp-script layout=container src="/examples/amp-script/hello-world.js">
+  <h1>Example: example.js</h1>
+  <amp-script layout=container src="/examples/amp-script/example.js">
```

Comment:
Can we name this something other than its grandparent folder name? Maybe `amp-script-demo.js`.